design/cli-install-changes.md

```diff
@@ -0,0 +1,156 @@
+# Proposal for a more intuitive CLI to install and configure Velero
+
+Currently, the Velero CLI tool has a `install` command that configures numerous major and minor aspects of Velero. As a result, the combined set of flags for this `install` command makes it hard to intuit and reason about the different Velero components. This document proposes changes to improve the UX for installation and configuration in a way that would make it easier for the user to discover what needs to be configured by looking at what is available in the CLI rather then having to rely heavily on our documentation for the usage. At the same time, it is expected that the documentation update to reflect these changes will also make the documentation flow easier to follow.
+
+This proposal prioritizes discoverability and self-documentation over minimalizing length or number of commands and flags.
+
+## Goals
```

Comment:
I think it might make sense to have a goal around having our install/config UX be ~consistent with the rest of the ecosystem. Related to this, it would probably be helpful (and this is something I'd like to do myself, regardless) to survey some other popular Kubernetes operators/extensions and see what their install UX looks like.

Reply:
Added to TODO list.